tables_io/io_layer.py

```diff
@@ -330,6 +416,23 @@ def writeDataframesToPq(dataFrames, filepath, **kwargs):
         _ = v.to_parquet("%s%s.pq" % (filepath, k), **kwargs)
 
 
+def readPqToDataframe(infile):
+    """
+    Reads a `pandas.DataFrame` object from an parquet file.
+
+    Parameters
+    ----------
+    infile: `str`
+        Path to input file
+
+    Returns
+    -------
+    df : `pandas.DataFrame`
+        The data frame
+    """
+    return pd.read_parquet(infile, engine='pyarrow')
```

Comment:
I haven't used this but it might do something similar to the HDF one?

I'm also wondering if we are repeating functionality from `astropy.table.Table` in these functions?

Reply:
Well, this version goes straight to pandas.   The h5py / astropy versions would have an intermediate step.